kubernetes/README.md

```diff
@@ -0,0 +1,5 @@
+# Kubernetes client libraries
+
+This package (and sub-packages) holds the client libraries for the kubernetes integration in
+the docker platform. Most of the code is currently generated but will evolved quickly in
+the next months.
```

Comment:
This comment will go stale quickly. Can we remove "will evolved quickly in the next months" ?

The plan is to keep this as generated code, right?

Reply:
We are discussing if we could write this code directly "by hand", instead of using a code generator which produced not-so-good code. But ok I'll remove the last part of the sentence.